backend/scripts/printful/downloadProductData.js

```diff
@@ -1,20 +1,61 @@
 const fs = require('fs')
 
+const difference = require('lodash/difference')
+
 const getProducts = require('./getProducts')
 const getProduct = require('./getProduct')
 const getProductIds = require('./getProductIds')
 
 const PrintfulURL = 'https://api.printful.com'
 
-async function downloadProductData({ OutputDir, printfulApi }) {
+/**
+ * Download's product data from Printful
```

Comment:
thanks for adding those jsdoc comments! super helpful.

Super nit: type. Downloads vs Download's